nodejs/week3/app.js

```diff
@@ -0,0 +1,65 @@
+const knex = require("knex")({
```

Comment:
You have knex config in app.js and database.js.
Please try to import this from database.js

Reply:
Hi @itsgauravsaxena,  

Thank you so much for your time reviewing my homework! 
I have fixed my code as per your suggestions

Pls let me know if I need to adjust something else or if I can merge it.